modules/tinymce/src/themes/silver/main/ts/ui/menus/item/structure/ItemStructure.ts

```diff
@@ -102,6 +104,16 @@ const renderItemDomStructure = (ariaLabel: Optional<string>): RawDomSchema => {
   };
 };
 
+const createLabel = (label: string): SimpleSpec => {
+  return {
+    dom: {
+      tag: 'label',
+      innerHtml: label
```

Comment:
Use `GuiFactory.text` instead of raw HTML.

- [ ] Use `GuiFactory.text` if plain text is to be rendered in alloy

Reply:
done